instrumented/integration/proguard-rules.pro

```diff
@@ -0,0 +1,3 @@
+
+-keep class ** { *; }
+-keepnames class com.datadog.android.sdk.** { *; }
```

Comment:
We keep everything (no shrinking), and prevent obfuscating the classes from the instrumented module